odtp/helpers/middleware.py

```diff
@@ -0,0 +1,72 @@
+import typer
+import requests
+import time
+import jwt
+
+
+"""
+This will be replace by handle by the gateway
+"""
+AUTH0_DOMAIN = 'http://localhost:8180/realms/myrealm/protocol/openid-connect/auth/device'
+AUTH0_CLIENT_ID = 'myclient'
+ALGORITHMS = ['RS256']
+CLIENT_SECRET = '95ZBagsIqS0ioTbwSwMFFwEgBXG1Fncn'
```

Comment:
This must be loaded from (secret) config files and not directly from the code. Otherwise, this exposes sensitive information and the code needs to be updated every time we deploy it somewhere else.